aequilibrae/paths/traffic_assignment.py

```diff
@@ -93,8 +94,9 @@ class TrafficAssignment(object):
         avg_skims = assigclass.results.skims # blended ones
         last_skims = assigclass._aon_results.skims # those for the last iteration
     """
+
     bpr_parameters = ["alpha", "beta"]
-    all_algorithms = ["all-or-nothing", "msa", "frank-wolfe", "fw", "cfw", "bfw"]
+    all_algorithms = ["all-or-nothing", "msa", "frank-wolfe", "fw", "cfw", "bfw", "bush-based", "bush-based-parallel"]
```

Comment:
Is the parallel version ready and coming with this PR?

Reply:
Good point, it's not ready and should be removed